### PR TITLE
Add about entry to settings menu

### PR DESCRIPTION
**Pull request by juliushaertl**
Implements #62 

image

@nextcloud/designers 

The help icon is changed in the server with https://github.com/nextcloud/server/pull/9411

### CODE REVIEW

File: lib/AppInfo/Application.php [hunk 1]
```diff
@@ -88,4 +93,18 @@ protected function registerNotificationNotifier() {
 			];
 		});
 	}
+
+	public function registerNavigation() {
+		$urlGenerator = $this->getContainer()->getServer()->getURLGenerator();
+		$l = $this->getContainer()->getServer()->getL10N('firstrunwizard');
+
+		$this->getContainer()->getServer()->getNavigationManager()->add([
```
Comment (nickvergessen) [inline lib/AppInfo/Application.php]: Please register via info.xml instead

Comment (skjnldsv) [reply lib/AppInfo/Application.php]: @nickvergessen where is this documented?

Comment (nickvergessen) [reply lib/AppInfo/Application.php]: https://nextcloudappstore.readthedocs.io/en/latest/developer.html#info-xml (search for navigation
Or https://docs.nextcloud.com/server/13/developer_manual/app/init.html?highlight=navigation

Comment (skjnldsv) [reply lib/AppInfo/Application.php]: @nickvergessen thanks :)

Comment (juliushaertl) [reply lib/AppInfo/Application.php]: Sure, I'll prepare a pull request for that.

**Pull request merged**